js/app.js

```diff
@@ -7,12 +7,10 @@ function Product(name,filePath){
     this.filePath=filePath;
     this.imgClickTotal=0;
     this.imgShownTotal=0;
-    products.push(this);
     this.previous = false;
+    products.push(this);
 }
-Product.prototype.chosen = function(){
-        this.imgClickTotal++;
-}
+
 //Images we will be showing
 new Product('bag','img/bag.jpg','bag');
```

Comment:
You are passing in three arguments, however your constructor only has two parameters. Thus, the last argument is unnecessary and is not being used.